haas/model.py

```diff
@@ -201,6 +154,7 @@ def start_console(self):
     # because we are not interested in the ouput of this command.
     @no_dry_run
     def stop_console(self):
+        return
```

Comment:
So, while I did suggest just doing this for now, I'm not really keen on merging it in like this. We could take a couple of approaches. The console stuff is being reworked in #483, so I don't think it makes sense to spend too much time messing with it on your end.

Perhaps the solution is to get that merged and then adjust this accordingly? We can still review the rest of it in the meantime.